includes/wikia/tests/GlobalTitleTest.php

```diff
@@ -3,9 +3,37 @@
 class GlobalTitleTest extends WikiaBaseTest {
 	private $wgDevelEnv;
 	private $wgDevelEnvName;
+	private $wikiUrlsMock = [
+		177 => 'http://community.wikia.com',
+		113 => 'http://en.memory-alpha.org',
+		490 => 'http://www.wowwiki.com',
+		1686 => 'http://spolecznosc.wikia.com',
+	];
 	
 	function setUp() {
 		parent::setUp();
+
+		$wgMemcMock = $this->getMockBuilder( 'MWMemcached' )
+			->disableOriginalConstructor()
+			->setMethods( [ 'get' ] )
+			->getMock();
+		$wgMemcMock->expects( $this->any() )
+			->method( 'get' )
+			->willReturn( null );
+		$this->mockGlobalVariable( 'wgMemc', $wgMemcMock );
```

Comment:
:+1: Why not move this code to ``disableMemCache`` helper in ``WikiaBaseTest`` class?